Lib/dis.py

```diff
@@ -32,25 +32,30 @@ def _try_compile(source, name):
     return c
 
 def dis(x=None, *, file=None, depth=None):
-    """Disassemble classes, methods, functions, generators,
-       asynchronous generators, coroutines, or code.
+    """Disassemble classes, methods, functions, other compiled objects, or code.
```

Comment:
Using both `other` and `or` here looks odd to me. Perhaps:

    """Disassemble classes, methods, functions, and other compiled objects.

I don't think it's necessary to specifically mention code objects in this first line - the fact that "other compiled objects" will include the output of a compile() call is a fairly obvious assumption.